openapi.yaml

```diff
@@ -1670,46 +1700,49 @@ tags:
     description: AML verification endpoints documentation.
   - name: applicants
     description: Applicant management endpoints documentation.
+  - name: attempts
+    description: Attempt management endpoints documentation.
 
 x-tagGroups:
   - name: Endpoints documentation
     tags:
       - Identity verifications
       - AML verifications
       - Applicants
+      - Attempts
```

Comment:
I think It would be better to order like this:
- applicants first
    - because it is common to all verifications types
    - also because it is the 1st ressource to be created
- identity verifications
    - because it is the 2nd created ;)
    - attempt inside, because it is related to identity-verifications -> identity-verifications/id/attempts
- AML verifications
    - because it is the 3rd

Reply:
done